utils/rtty/Makefile

```diff
@@ -31,7 +31,7 @@ define Package/rtty/default
   CATEGORY:=Utilities
   SUBMENU:=Terminal
   TITLE:=A reverse proxy WebTTY
-  DEPENDS:=+libprotobuf-c
+  DEPENDS:=+libprotobuf-c +@BUSYBOX_CUSTOM +@BUSYBOX_CONFIG_LOGIN
```

Comment:
If this will cause busybox login to be selected by default for all buildbot builds, then this is not acceptable.

Reply:
If so, what can I do?

Reply:
OK.